components/SearchBar/index.js

```diff
@@ -29,22 +29,17 @@ const SearchButton = styled.button`
     transform: scale(0.95);
   }
 `;
-export default function SearchBar({
-  searchTerm,
-  onSearchTermChange,
-  onSearch,
-}) {
+export default function SearchBar({ onSearch }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const searchTerm = event.target.elements.searchTerm.value;
+    onSearch(searchTerm);
+  }
+
```

Comment:
very nice! splitting the submit functionality into two functions (onSearch from the outside and handleSubmit here) is very clean ✨